common/rabbitmq/templates/service.yaml

```diff
@@ -3,14 +3,7 @@ kind: Service
 metadata:
   name: {{ template "fullname" . }}
   labels:
-    app: {{ template "fullname" . }}
```

Comment:
Below the necessary change to service selector is missing, see
```
spec:
  selector:
    app: {{ template "fullname" . }}
```

Reply:
thanks for the hint